trademark.md

```diff
@@ -0,0 +1,72 @@
+Dask Trademark Policy
+=====================
+
+This document describes the policy governing the use of the "Dask" trademark,
+which is owned by NumFOCUS and administered on behalf of the Dask project.  Any
+use of the Dask trademark commercially or non-commercially must be in
+accordance with the policy described in this document unless specific written
+permission is obtained from NumFOCUS.
+
+Usage and modification of the Dask *software* is governed by the
+[license](https://github.com/dask/dask/blob/main/LICENSE.txt), which describes
+permissable behavior under copyright law.  Trademarks are a distinct form of
+intellectual property, and so the guidelines in this document apply only to
+the "Dask" trademark and logo, but not to the software itself.
+
+Intent
+------
+
+Our goal is to enable the Dask trademark and logo to be used with minimal
+restrictions by both commercial and non-commercial entities, but we do *not*
+want the Dask trademark to be used:
+
+* to cause confusion about whether a project, product, or service is endorsed
+  by the Dask community in some way.
+* in a way that allows an entity to misleadingly imply ownership of Dask or
+  influence over the Dask project.
+
+Permissable usage
+-----------------
+
+The Dask trademark is always subject to "nominative use rules" which allow the
+usage of the trademark for descriptive purposes that does not suggest
+sponsorship or endorsement by the trademark owner.  You may freely describe
+that your product or service "uses Dask" or is "powered by Dask," and you
+may use the unaltered Dask logo to indicate this without prior approval.
+(Please see the [Dask Brand Guide](https://www.dask.org/brand-guide) for more
+details about how to use the Dask logo.)
+
+Open source projects may also use the Dask name as part of the name of their
+project (Ex: `dask-cuda`) so long as it satisfies *all* the following
+criteria:
+
+* The project is open source with a liberal license.
+* The project is related to Dask.
+* The project name should not be overly grand in claiming namespace.
+* The project should not make subjective claims in the name itself.
+
+Usage of the "Dask" trademark should also be consistent with the [Dask Code of Conduct](https://github.com/dask/governance/blob/main/code-of-conduct.md).
+
+Non-permissible usage
+---------------------
+
+The Dask trademark may *not* be used in a commercial product or company name.  Examples of disallowed uses include:
+
+* Dask Pro
+* Dask Enterprise
+* The Dask Company
+* CompanyName Dask
+
+Company and product logos may not derive from the Dask logo in a way that would cause brand confusion.
+
+Where to get more information
+-----------------------------
+
+If you are unsure whether your planned usage of the Dask trademark is allowed,
+or feel that you need specific written approval for your situation, please
+contact NumFOCUS at *INSERT CONTACT EMAIL HERE*.
```

Comment:
```suggestion
contact NumFOCUS at info@dask.org.
```

Reply:
I will make this change, but rephrase to say "contact the Dask maintainers at info@dask.org."